docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -23,3 +23,38 @@
 	| `MRI_alias`  | MRI specific shortname. Limited to 4 characters  |                                                                                              |
 	| `Account`    |                                                  | Deprecated                                                                                              |
 	| `Study_site` | Does this site recruit candidates?             | This field will affect the appearance of the center in dropdowns in several modules of LORIS |
+
+
+- Table: `Project`
+ 
+ 	*This table stores the list of projects configured for the study.*
+ 
+	|         Field        |                  Description                  |                                        Notes                                                 |
+	|:--------------------:|:---------------------------------------------:|:--------------------------------------------------------------------------------------------:|
+	| `ProjectID`          | Identifier of the project                     | Avoid setting this field explicitly when inserting data, it auto increments.                 |
+	| `Name`               | Full name of the project                      |                                                                                              |
+	| `recruitmentTarget`  | Expected number of candidates to be recruited |                                                                                              |
+
+
+- Table: `subproject`
+ 
+ 	*This table stores the list of subprojects configured for the study.*
+ 
+	|        Field       |                  Description                  |                                     Notes                                    |
+	|:------------------:|:---------------------------------------------:|:----------------------------------------------------------------------------:|
+	| `SubprojectID`     | Identifier of the subproject                  | Avoid setting this field explicitly when inserting data, it auto increments. |
+	| `title`            | Name of the subproject                        |                                                                              |
+	| `useEDC`           | Use the Expected date Of Confinement          |                                                                              |
+	| `WindowDifference` |                                               | Deprecated ??                                                                |
```

Comment:
is there an answer whether this field is deprecated? Just wanted to make a comment so we don't lose track of this and avoid having '??' in the final WIKI ;)

Reply:
I have no idea what it does !!

@driusan @xlecours @everyoneelse does anyone know ?

Reply:
Didn't someone explain how this worke din the LORIS meeting when I asked about `Visit_windows`?